buildpacks/buildpacks-v3.yaml

```diff
@@ -1,134 +1,132 @@
-      mountPath: /layers
+    - name: analyze
+      image: $(inputs.params.BUILDER_IMAGE)
+      imagePullPolicy: Always
+      command: ["/cnb/lifecycle/analyzer"]
+      args:
+        - "-layers=/layers"
+        - "-helpers=$(inputs.params.USE_CRED_HELPERS)"
+        - "-group=/layers/group.toml"
+        - "$(outputs.resources.image.url)"
+      volumeMounts:
+        - name: layers-dir
+          mountPath: /layers
 
-  - name: build
-    image: $(inputs.params.BUILDER_IMAGE)
-    imagePullPolicy: Always
-    command: ["/lifecycle/builder"]
-    args:
-    - "-app=/workspace/source"
-    - "-layers=/layers"
-    - "-group=/layers/group.toml"
-    - "-plan=/layers/plan.toml"
-    volumeMounts:
-    - name: "layers-dir"
-      mountPath: /layers
+    - name: restore
+      image: $(inputs.params.BUILDER_IMAGE)
+      imagePullPolicy: Always
+      command: ["/cnb/lifecycle/restorer"]
+      args:
+        - "-group=/layers/group.toml"
+        - "-layers=/layers"
+        - "-cache-dir=/cache"
+      volumeMounts:
+        - name: $(inputs.params.CACHE)
+          mountPath: /cache
+        - name: layers-dir
+          mountPath: /layers
 
-  - name: export
-    image: $(inputs.params.BUILDER_IMAGE)
-    imagePullPolicy: Always
-    command: ["/lifecycle/exporter"]
-    args:
-    - "-app=/workspace/source"
-    - "-layers=/layers"
-    - "-helpers=$(inputs.params.USE_CRED_HELPERS)"
-    - "-group=/layers/group.toml"
-    - "$(outputs.resources.image.url)"
-    volumeMounts:
-    - name: "layers-dir"
-      mountPath: /layers
+    - name: build
+      image: $(inputs.params.BUILDER_IMAGE)
+      imagePullPolicy: Always
+      command: ["/cnb/lifecycle/builder"]
+      args:
+        - "-app=/workspace/source/$(inputs.params.SOURCE_SUBPATH)"
+        - "-layers=/layers"
+        - "-group=/layers/group.toml"
+        - "-plan=/layers/plan.toml"
+      volumeMounts:
+        - name: layers-dir
+          mountPath: /layers
 
-  - name: cache
-    image: $(inputs.params.BUILDER_IMAGE)
-    imagePullPolicy: Always
-    command: ["/lifecycle/cacher"]
-    args:
-    - "-layers=/layers"
-    - "-group=/layers/group.toml"
-    - "-path=/cache"
-    volumeMounts:
-    - name: "$(inputs.params.CACHE)"
-      mountPath: /cache
-    - name: "layers-dir"
-      mountPath: /layers
+    - name: export
+      image: $(inputs.params.BUILDER_IMAGE)
+      imagePullPolicy: Always
+      command: ["/cnb/lifecycle/exporter"]
+      args:
+        - "-app=/workspace/source/$(inputs.params.SOURCE_SUBPATH)"
+        - "-layers=/layers"
+        - "-helpers=$(inputs.params.USE_CRED_HELPERS)"
+        - "-group=/layers/group.toml"
+        - "-cache-dir=/cache"
+        - "$(outputs.resources.image.url)"
+      volumeMounts:
+        - name: layers-dir
+          mountPath: /layers
+        - name: $(inputs.params.CACHE)
+          mountPath: /cache
 
   volumes:
-  - name: empty-dir
-    emptyDir: {}
-  - name: layers-dir
-    emptyDir: {}
+    - name: empty-dir
```

Comment:
@natalieparellano should `empty-dir` also use `$(inputs.params.CACHE)` placeholder?

Reply:
I don't believe so - `CACHE` will default to the `empty-dir` if no value is provided.